apps/E2E/src/Button/specs/Button.spec.win.ts

```diff
@@ -30,18 +26,35 @@ describe('Button Accessibility Testing', () => {
     await ButtonPageObject.scrollToTestElement();
   });
 
-  it('Button - Validate accessibilityRole is correct', async () => {
-    await expect(await ButtonPageObject.getAccessibilityRole()).toEqual(BUTTON_A11Y_ROLE);
+  it('Validate "accessibilityRole" value is correct, propagates to "ControlType" element attribute.', async () => {
```

Comment:
I think it'd be useful to mention what control type we're expecting so I'd reword this as "Validate "accessibilityRole" value defaults to "Button" ControlType." or something similar.